tests/django/test_views.py

```diff
@@ -99,6 +99,19 @@ def test_fails_when_not_sending_query():
         assert e.value.args == ("No GraphQL query found in the request",)
 
 
+def test_fails_when_request_body_has_invalid_json():
+    factory = RequestFactory()
+
+    request = factory.post(
+        "/graphql/", "definitely-not-json-string", content_type="application/json"
+    )
+
+    with pytest.raises(SuspiciousOperation) as e:
+        GraphQLView.as_view(schema=schema, graphiql=False)(request)
+
+    assert e.value.args == ("Unable to parse request body as JSON",)
```

Comment:
Here with the tests I discovered the following behaviour, which I am fairly sure is not the intention.

### assert in the context ( as above on line 99 ):
The test **only checks whether the exception is raised or not**, the assertion of the exception message is ignored.

```python
with pytest.raises(SuspiciousOperation) as e:
    GraphQLView.as_view(schema=schema, graphiql=False)(request)
    assert e.value.args == ("blablabla",)  # test passes no matter what the string is
```


### If I move the assert outside the context, then it works as expected.